docs/UserGuide.adoc

```diff
@@ -17,25 +18,24 @@ endif::[]
 By: `Team T09-B3`      Since: `Feb 2018`      Licence: `MIT`
 
 == Introduction
-
 === About the App
 
 ProgressChecker is for *students* who *prefer to use a desktop app* to keep track of their learning progress throughout the certain module. (Current version is available for CS2103/T) +
 
 For the current version, students can add their teammates into the app. They can create a new task list through google task. By default, app will display all the link:DeveloperGuide.adoc#Learning-Outcomes[Learning Outcomes] for this week in the task list. Students can use this task list to track their weekly homework and the progress of the project.
```

Comment:
Not sure if introduction should be using 'You' language as well.

But perhaps consider using a statement like ```For the current version, you can add your teammates details into ProgressChecker. ...``` to

1. be more concise on what adding a teammate means and
2. to make the guide more personal to our software by frequently using the name ProgressChecker instead of the app